app/config/services.php

```diff
@@ -146,3 +329,53 @@
 $di->set('acl', function () {
     return new Acl();
 });
+
+
+/*
+ *  Translation application use gettext
+ **/
+$di->set(
+    'translation',
+    function () use ($di) {
+        $translation = new Gettext(
+            [
+                'locale' => $di->get('config')->language,
+                'directory' => ROOT_DIR . '/apps/lang',
+                'defaultDomain' => 'messages',
+
+            ]
+        );
+        return $translation;
+    },
+    true
+);
+
+/**
+ * Translation function call anywhere
+ *
+ * @param $string
+ *
+ * @return mixed
+ */
+function t($string)
+{
+    $translation = DI::getDefault()->get('translation');
+    return $translation->_($string);
+}
+
+/*
+ *  Phalcon Debugger
+ **/
+if ($config->application->debug) {
+    (new \Phalcon\Debug)->listen();
+
+    function dpm($object, $kill = true)
+    {
+        echo '<pre style="text-aling:left">';
+        print_r($object);
+        if ($kill) {
+            die('END');
```

Comment:
I think that it didn't make it in because it was a massive set of changes.  Vokuro is a somewhat of a documentation project.  sergeyklay asked him about his intentions to document his changes and he shied away from responding to that inquery.  Also its really better to bring in the changes in smaller batches.